### PR TITLE
Normalize Alcremie Day/Night Cycle

### PR DESCRIPTION
**Pull request by Razorflamekun**
This pull request is meant to normalize the Day/Night cycle in the Battle Cafe to make it consistent with the rest of the game.

### CODE REVIEW

**Comment by Jaaslet** [comment]
Night should start at 18, right?

**Comment by Razorflamekun** [comment]
Yeah, did I set it to something else?

File: src/scripts/towns/BattleCafe.ts [hunk 1]
```diff
@@ -77,9 +77,9 @@ class BattleCafeController {
             (new PokemonItem('Milcery (Cheesy)', 0)).gain(1);
             return;
         }
-        if (curHour == 19 && !clockwise && spinTime > 10) {
+        if (curHour == 17 && !clockwise && spinTime > 10) {
             spin = GameConstants.AlcremieSpins.at7Above10;
-        } else if (curHour >= 5 && curHour < 19) { // Is day
+        } else if (curHour >= 6 && curHour < 17) { // Is day
```
Comment (Jaaslet) [inline src/scripts/towns/BattleCafe.ts:82]: It's day if current hour is less than 17

Comment (Razorflamekun) [reply src/scripts/towns/BattleCafe.ts:82]: Yes, that should be the case, as dusk is it's own time period, and then night starts at 18, so if this is set to 17, then it should make it day from 6-17, Dusk from 17-18, and Night from 18-19, right?

Comment (Razorflamekun) [reply src/scripts/towns/BattleCafe.ts:82]: Did I miss something in my changes?  I'm new to coding.

Comment (Jaaslet) [reply src/scripts/towns/BattleCafe.ts:82]: `<` is less than. So `<17` means as long as the current hour is less than 17. So when the clock turns 17, it's no longer true

Comment (Qwertypop04) [reply src/scripts/towns/BattleCafe.ts:82]: > Yes, that should be the case, as dusk is it's own time period, and then night starts at 18, so if this is set to 17, then it should make it day from 6-17, Dusk from 17-18, and Night from 18-19, right?

Dusk is the last hour of day. So from 17-18, it should be day, and also dusk

Comment (Razorflamekun) [reply src/scripts/towns/BattleCafe.ts]: I've fixed the         } else if (curHour >= 6 && curHour < 17) { // Is day to be         } else if (curHour >= 6 && curHour < 18) { // Is day

now. This should fix the issues, correct?

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**